app/helpers/messages_helper.rb

```diff
@@ -1,2 +1,14 @@
 module MessagesHelper
+
+  def personal_address
+    current_user.name? ? current_user.name : current_user.email
+  end
+
+  def timestamp_for_messages(message)
+    if message.created_at > Time.now - 1.hour
+        time_ago_in_words(message.created_at) +' ago'
```

Comment:
If you can update your text editor, you should set 2 spaces globally. Looks like this is 4 spaces